tools/src/main/scala/caliban/tools/Codegen.scala

```diff
@@ -57,15 +57,16 @@ object Codegen {
                                         scalarMappings
                                       )
                                   }
-      formatted                <- if (enableFmt) Formatter.format(code, arguments.fmtPath) else Task.succeed(code)
+      formatted                <- if (enableFmt) Formatter.format(code, arguments.fmtPath) else ZIO.succeed(code)
       paths                    <- ZIO.foreach(formatted) { case (objectName, objectCode) =>
                                     val path =
                                       if (splitFiles) s"${arguments.toPath.reverse.dropWhile(_ != '/').reverse}$objectName.scala"
                                       else arguments.toPath
 
-                                    Task.blocking(
-                                      Task(new PrintWriter(new File(path)))
-                                        .acquireReleaseWith(q => UIO(q.close()), pw => Task(pw.println(objectCode)))
+                                    ZIO.blocking(
```

Comment:
I saw there was `attemptBlocking`, I guess we could use it here?

Reply:
I dunno, is it better to wrap acquire, release and use all with `attemptBlocking` or wrap the whole task with `blocking`?

Reply:
Ah yeah you're right. It's good as it is!